pages/resources/collectibles.mdx

```diff
@@ -0,0 +1,802 @@
+| --------------------- | ------- | --------------------------------------------------------------- |
+| type                  | integer | The [collectibles marketing type](#collectibles-marketing-type) |
+| title                 | string  | The title of the coachmark                                      |
+| body                  | string  | The body text of the coachmark                                  |
+| asset_dark            | string  | The URL of the dark mode coachmark image                        |
+| asset_light           | string  | The URL of the light mode coachmark image                       |
+| version               | integer | The version of the coachmark                                    |
+| ref_target_background | object  | The target background configuration                             |
+
+###### Collectible Style Structure
+
+| Field             | Type           | Description                   |
+| ----------------- | -------------- | ----------------------------- |
+| background_colors | array[integer] | An array of background colors |
+| button_colors     | array[integer] | An array of button colors     |
+| confetti_colors   | array[integer] | An array of confetti colors   |
+
+###### Collectible User Discount Structure
+
+| Field       | Type               | Description               |
+| ----------- | ------------------ | ------------------------- |
+| amount      | integer            | How much the discount is  |
+| discount_id | snowflake          | The discount ID           |
+| expires_at  | ?ISO8601 timestamp | When the discount expires |
+
+###### Gift Eligibility Structure
+
+| Field | Type    | Description                                           |
+| ----- | ------- | ----------------------------------------------------- |
+| valid | boolean | Whether the recipient is eligible to receive the gift |
+
+###### Variant Return Style
+
+| Value | Name                | Description                                        |
+| ----- | ------------------- | -------------------------------------------------- |
+| 1     | INDIVIDUAL_PRODUCTS | Variants should be returned as individual products |
+| 2     | VARIANTS_GROUP      | Variants should be returned as a group             |
+
+###### Collectibles Shop Home Config
+
+| Value                  | Name                   | Description                                        |
+| ---------------------- | ---------------------- | -------------------------------------------------- |
+| default                | DEFAULT                | The default shop home configuration                |
+| default_with_orb_shelf | DEFAULT_WITH_ORB_SHELF | The default shop home configuration with orb shelf |
+| orb_tab                | ORB_TAB                | The orb tab shop home configuration                |
+| summer_sale_takeover   | SUMMER_SALE_TAKEOVER   | The summer sale takeover shop home configuration   |
+
+###### Collectibles Shop Home Tab
```

Comment:
why is this in the marketing object. move under the Get Collectibles Shop endpoint's query params